### ISSUE TITLE
xls package is broken

### ISSUE THREAD

**Issue by proppy** [hdl/conda-eda#353]
https://github.com/hdl/conda-eda/actions/runs/5944971903/job/16123228091

**Comment by proppy** [comment]
seems to be related to some of the assumption z3 makes about the c++ toolchain:
```
2023-08-22T23:37:23.0766098Z [32;1m23:37:23[0m | external/z3/src/qe/qe_vartest.h:26:38: error: no template named 'unary_function' in namespace 'std'; did you mean '__unary_function'?
2023-08-22T23:37:23.0767473Z [32;1m23:37:23[0m | class is_variable_proc : public std::unary_function<expr*,bool> {
2023-08-22T23:37:23.0767977Z [32;1m23:37:23[0m |                                 ~~~~~^
2023-08-22T23:37:23.0768714Z [32;1m23:37:23[0m | external/llvm_toolchain_llvm/bin/../include/c++/v1/__functional/unary_function.h:46:1: note: '__unary_function' declared here
2023-08-22T23:37:23.0769282Z [32;1m23:37:23[0m | using __unary_function = __unary_function_keep_layout_base<_Arg, _Result>;
2023-08-22T23:37:23.0769709Z [32;1m23:37:23[0m | ^
2023-08-22T23:37:23.0770069Z [32;1m23:37:23[0m | 1 error generated.
```

**Comment by proppy** [comment]
related to https://github.com/Z3Prover/z3/pull/1880

**Comment by proppy** [comment]
fixed with #356
